extensions/ti-debpkgs.sh

```
@@ -0,0 +1,33 @@
+function extension_prepare_config__add_packages() {
+	if [[ ${#TI_PACKAGES[@]} -gt 0 ]] ; then
+		add_packages_to_rootfs "${TI_PACKAGES[@]}"
+	fi
+}
+
+function custom_apt_repo__install_ti_packages() {
+	# Fetch valid suites from TI repo
+	valid_suites=($(curl -s "https://api.github.com/repos/TexasInstruments/ti-debpkgs/contents/dists" | jq -r '.[].name'))
```

Shellcheck (0.10.0)

[warning] Prefer mapfile or read -a to split command output (or quote to avoid splitting).

(SC2207)